### DISCUSSION TITLE
Login screen on login

### DISCUSSION THREAD

**Discussion by rpedrica** [poweradmin/poweradmin#430] [Q&A]
Hi poweradmin,

I've installed poweradmin per the guides/docs and everything seem in order, however, when I try to login, I get taken back to the login screen. The apache logs indicated login was successful and I can confirm this by logging in with a bad user/pwd and then I get an error about auth failed. With correct details, there are not auth errors + with log, seems to indicate that login is successful but I'm still at the login screen. Any assistance appreciated.

Thanks, Robby

**Comment by edmondas** [comment]
@rpedrica Check the latest release, and let me know if the issue persists. Verify that php-intl is installed on the system.
